lib/monitoring/checkers.go

```diff
@@ -178,8 +181,8 @@ func addToMaster(node agent.Agent, config *Config, etcdConfig *monitoring.ETCDCo
 	}
 	node.AddChecker(NewVersionCollector())
 	node.AddChecker(monitoring.NewDNSChecker([]string{
-		"leader.telekube.local",
-	}))
+		"leader.telekube.local.",
+	}, config.LocalNameservers...))
```

Comment:
I wonder if this should detect / skip the check if config is missing? I think the Checker falls back to loading resolv.conf if the nameservers are empty.